app/main/controllers/activity/RTMediaBuddyPressActivity.php

```diff
@@ -847,4 +848,33 @@ public function rtm_bp_activity_permalink_access( $args ) {
 		return $has_access;
 
 	}
+
+	/**
+	 * Makes the comments hidden (private) if the parent comment's
+	 * privacy is set to private.
+	 *
+	 * @param string $comment_id Activity id of the comment.
+	 * @param array  $r          Array of arguments.
+	 */
+	public function rtm_check_privacy_for_comments( $comment_id, $r ) {
```

Comment:
Add `! empty()` check for `$commentd_id` and `$r`. So it prevent any notices/warnings.